hedera-node/hedera-app/src/main/java/com/hedera/node/app/throttle/ThrottleAccumulator.java

```diff
@@ -531,7 +542,8 @@ public void applyGasConfig() {
         gasThrottle = new GasLimitDeterministicThrottle(capacity);
 
         log.info(
-                "Resolved consensus gas throttle -\n {} gas/sec (throttling {})",
+                "Resolved {} gas throttle -\n {} gas/sec (throttling {})",
+                throttleType,
```

Comment:
Is this the only place we use the `ThrottleType.type()`? If so, I think we can remove it and use the enum's name directly. It is just us who are reading the logs.

Reply:
We use it for another log as well but it makes sense to use the enum's name, yes 👍